release/ray_release/cluster_manager/minimal.py

```diff
@@ -224,7 +227,7 @@ def create_cluster_compute(self, _repeat: bool = True):
                         dict(
                             name=self.cluster_compute_name,
                             project_id=self.project_id,
-                            config=self.cluster_compute,
+                            config=self.cluster_compute_with_extra_tags,
```

Comment:
I'm not a fan of these magic properties. Can we just have an explicit method that adds the tags to the cluster compute?

Reply:
I think it should be like this:

1. Fetch tags to set, put in e.g. `tags` variable
2. Fetch cloud type, e.g. AWS/GCP
3. Call something like `annotate_cluster_compute(cluster_compute, cloud_type, tags)`

Also I think this should happen in `set_cluster_compute` so that the hash is built using the tags (otherwise we may end up using an existing cluster compute with wrong tags)